recipe/cmake_test/CMakeLists.txt

```diff
@@ -1,4 +1,45 @@
 project(cf_dummy LANGUAGES C CXX)
 cmake_minimum_required(VERSION 3.12)
+
+option(WITH_TORCH_PYTHON "Build with torch_python" OFF)
+
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 CONFIG REQUIRED)
```

Comment:
Can you move this behind `if(WITH_TORCH_PYTHON)` please? That way we can leave out `pybind11` as a test dependency for `libtorch`.